### DISCUSSION TITLE
llama-3-70B and PHI-3 - instruction template, chat template, fine tune.

### DISCUSSION THREAD

**Discussion by Zati1980** [oobabooga/text-generation-webui#5973] [Q&A]
I have questions about these 2 new models of LLM.
It is possible to load them to Oobabooga what loaders should be used? 
How the template for instruction and chat should look like?
How the fine-tuning of these 2 models should look like for the hardware for example 0 GPU NVIDIA 3090 RTX 24 VRAM, 1 NVIDIA 4080 RTX 18 GB VRAM, 164 GB RAM, proc  13th Gen Intel(R) Core(TM) i9-13900KS   3.20 GHz? , that answers be faster if that is possible? 
Sorry, I have read so many pages in 3 weeks and there is no comprehensive gathered explanation for all. I'm trying to gather all the information for that comprehensive document with examples.

**Comment by michusx** [comment]
Can anyone help in the above topics?
In my case I'm specially interested in configuration of text-generation-webui to use PHI-3 in 128k-instruct version. Currently I managed to run it but when answering it falls into endless loop until context is exchausted. I've read on Huggingface in answers to use extra stop tokens: 32000, 32001 and 32007 but I'm not sure where to put them in GUI. Is it "Custom stopping strings" field in Parameters/Generation a correct place for this? And should it be a list of corresponding strings like: 
"<|endoftext|>"
"<|assistant|>"
"<|end|>"
or something else?
